src/analytics/Events.tsx

```diff
@@ -430,6 +430,8 @@ export enum FiatExchangeEvents {
   cico_spend_select_provider_back = 'cico_spend_select_provider_back',
   cico_non_celo_exchange_send_bar_continue = 'cico_non_celo_exchange_send_bar_continue', // When send bar is tapped from cash out for cUSD & cEUR
   cico_celo_exchange_send_bar_continue = 'cico_celo_exchange_send_bar_continue', // When withdraw bar is tapped from cash out CELO
+
+  cico_select_currency_back = 'cico_select_currency_back',
```

Comment:
nit: could you add a comment? I know we haven't been very thorough about this, but it helps tremendously when checking analytics behavior.